### ISSUE TITLE
Examples

### ISSUE THREAD

**Issue by jimjones26** [UnwrittenFun/pims#11]
Sorry if this is the wrong place for this, but I was wondering if you could do a few examples to show how to implement this. I have been using thinkagain for describing my rethinkdb models but I would like to move away from that and use typescript interfaces instead (a search that brought me here).

**Comment by UnwrittenFun** [comment]
Yeah I need to work on adding, well.. any amount of documentation. I'll try and fit some time in to make some examples over the next few days 😄
